### PR TITLE
Announce 4.56

### PR DESCRIPTION
**Pull request by fredemmott**

### CODE REVIEW

File: _posts/2020-05-04-hhvm-4.56.markdown [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+we are changing our distribution support lifecycle, starting with nightly
+builds and the next release:
+
+- MacOS: we aim to support the latest stable release - currently Catalina.
+  When a new version of MacOS is released, we plan to support both for 6
+  months.
+- Debian: we aim to support Debian stable. When a new version is promoted to
+  stable, we plan to support both for 6 months.
+- Ubuntu LTS: we aim to support the current LTS release (20.04); when a new
+  Ubuntu LTS is released, we plan to support both for the next month.
```
Comment (jjergus) [inline _posts/2020-05-04-hhvm-4.56.markdown:50]: Shouldn't it be 6 months here as well?

Comment (fredemmott) [reply _posts/2020-05-04-hhvm-4.56.markdown:50]: good catch on both :)

Comment (fredemmott) [reply _posts/2020-05-04-hhvm-4.56.markdown:50]: Ah ,yeah, I did diff from HHVM-4.55 to nightly, not from the previous week, so commit list is good. This should be 6 months

**Comment by fredemmott** [comment]
Announcement ready, fun with mac builds due to homebrew change.

**Comment by fredemmott** [comment]
Squashed as 03a190470b0e09cfcf94420a272974f6f8d1b324

**Pull request closed without merging**